includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -17,15 +17,18 @@ class DesignSystemCommunityHeaderModel extends WikiaModel {
 	private $wikiLocalNavigation = null;
 
 	public function __construct( string $langCode ) {
-		global $wgCityId;
+		global $wgCityId, $wgFandomCreatorCommunityId;
 
 		parent::__construct();
 
 		$this->productInstanceId = $wgCityId;
 		$this->langCode = $langCode;
 		$this->themeSettings = new ThemeSettings( $wgCityId );
 		$this->settings = $this->themeSettings->getSettings();
-		$this->mainPageUrl = wfProtocolUrlToRelative( Title::newMainPage()->getFullURL() );
+		$this->mainPageUrl = empty( $wgFandomCreatorCommunityId )
```

Comment:
@xkxd IMO we should move overriding of mainPageUrl to the hook where previously mainpage url was set to /

Reply:
mainPageUrl is used in `workmark` as well